### PR TITLE
Setup structure for Bookstore app

### PR DESCRIPTION
**Pull request by fibini**
-Add [React Router] for the app's navigation:
-Display the list of books with the Remove button (no functionality yet).
-Create a component called Book for displaying a single book (receive a title and an author as a prop) and reuse it in a component that displays a list of books.
-Create a separate component for form with inputs for a title and an author 
-Categories display a button "Check status" only.

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Form = () => (
+  <div>
+    <h2>ADD NEW BOOK</h2>
+    <input placeholder="Book title" />
+    <input placeholder="Author" />
+    <button type="submit">ADD BOOK</button>
```
Comment (Ismailco) [inline src/components/Form.js]: - [ ] This section should render a form with two inputs and a button to submit the form (the button is just a placeholder for now), kindly wrap the inputs and submit button with a form tag so your form can be submitted in the future when adding the functionality

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @fibini, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Using the Microverse template

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Modifying the README so it is more professional, and customized for your project specifically

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.js (hunk 1 above)
Comment (rloterh) [reply src/components/Form.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Fabien,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉🌟 

## Highlights 👍
- Correctly used React Router
- Correct;u implemented React components


<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**